tests/test_views.py

```diff
@@ -369,14 +369,14 @@ def test_context_with_post_and_field_error(self, client):
             data
         )
         expected_error = {
-            'scope': [u'Select a valid choice. 2 is not one of the available choices.']
+            'scope': ['Select a valid choice. 2 is not one of the available choices.']
         }
 
         assert response.context['form_errors'] == expected_error
         assert response.context['summary_list'] is None
         assert response.context['scope_form'].is_valid() is False
-        assert sorted(json.loads(response.context['json_data'])) == sorted(
-            [[key, [str(data[key])]] for key in data])
+        assert sorted(response.context['json_data']) == sorted(
+            [(key, [str(data[key])]) for key in data])
```

Comment:
Here and other places where tests were changed to accommodate removing the `json.dumps` being removed above, will need to be changed back.